dotnetv3/EventBridge Scheduler/Scenarios/SchedulerWorkflow.cs

```diff
@@ -0,0 +1,655 @@
+                }
+            }
+
+            Console.WriteLine("Waiting for CloudFormation stack creation to complete...");
+            await Task.Delay(retryDelay);
+            retryCount++;
+        }
+
+        _logger.LogError("Timed out waiting for CloudFormation stack creation to complete.");
+        return false;
+    }
+
+    /// <summary>
+    /// Retrieves the output values from the CloudFormation stack.
+    /// </summary>
+    /// <param name="stackId">The ID of the CloudFormation stack.</param>
+    private static async Task<bool> GetStackOutputs(string stackId)
+    {
+        try
+        {
+            var describeStacksRequest = new DescribeStacksRequest { StackName = stackId };
+
+            var describeStacksResponse =
+                await _amazonCloudFormation.DescribeStacksAsync(describeStacksRequest);
+
+            if (describeStacksResponse.Stacks.Count > 0)
+            {
+                var stack = describeStacksResponse.Stacks[0];
+                _roleArn = GetStackOutputValue(stack, "RoleARN");
+                _snsTopicArn = GetStackOutputValue(stack, "SNStopicARN");
+                return true;
+            }
+            else
+            {
+                _logger.LogError($"No stack found for stack outputs: {stackId}");
+                return false;
+            }
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(
+                ex, $"Failed to retrieve CloudFormation stack outputs: {stackId}");
+            return false;
+        }
+    }
+
+    /// <summary>
+    /// Get an output value by key from a CloudFormation stack.
+    /// </summary>
+    /// <param name="stack">The CloudFormation stack.</param>
+    /// <param name="outputKey">The key of the output.</param>
+    /// <returns>The value as a string.</returns>
+    private static string GetStackOutputValue(Stack stack, string outputKey)
+    {
+        var output = stack.Outputs.First(o => o.OutputKey == outputKey);
+        var outputValue = output.OutputValue;
+        Console.WriteLine($"Stack output {outputKey}: {outputValue}");
+        return outputValue;
+    }
+
+    /// <summary>
+    /// Creates a one-time schedule to send an initial event.
+    /// </summary>
+    /// <returns>True if the one-time schedule was created successfully.</returns>
+    public static async Task<bool> CreateOneTimeSchedule()
+    {
+        var scheduleName =
+            PromptUserForResourceName("Enter a name for the one-time schedule:");
+
+        Console.WriteLine($"Creating a one-time schedule named '{scheduleName}' to send an initial event in 1 minute...");
+        try
+        {
+            // Create a one-time schedule with a flexible time
+            // window set to delete after completion.
+            // You may also set a timezone instead of using UTC.
+            var scheduledTime = DateTime.UtcNow.AddMinutes(1).ToString("s");
+
+            await _schedulerWrapper.CreateScheduleAsync(
```

Comment:
Same comment as above about honoring bool return values.

Reply:
I updated to return it to the parent, but at the highest level if it fails, the user will be notified, but the rest of the workflow can continue.